internal/controller/datadogagent/component/agent/default.go

```diff
@@ -397,23 +407,9 @@ func envVarsForSecurityAgent(dda metav1.Object) []corev1.EnvVar {
 	return append(envs, commonEnvVars(dda)...)
 }
 
-func envVarsForOtelAgent(dda metav1.Object) []corev1.EnvVar {
-	envs := []corev1.EnvVar{
-		// TODO: add additional env vars here
-	}
-
-	return append(envs, commonEnvVars(dda)...)
-}
-
 func volumeMountsForInitConfig() []corev1.VolumeMount {
 	return []corev1.VolumeMount{
-		common.GetVolumeMountForLogs(),
```

Comment:
Why delete initContainer configs, is this even in otel agent code path?

Reply:
This was a mistake, meant to remove for otel agent voluments. undid the change